carbonj.service/build.gradle

```diff
@@ -190,44 +190,51 @@ release {
-    compile group: 'io.dropwizard.metrics', name: 'metrics-jvm', version: '4.2.13'
-    compile group: 'io.dropwizard.metrics', name: 'metrics-graphite', version: '4.2.13'
-
-    testCompile group: 'junit', name: 'junit', version: '4.13.2'
-    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
-
-    compile (group:"org.springframework.boot", name:"spring-boot-starter-logging", version:"2.7.5") {
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.19.0'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.19.0'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: '2.19.0'
+    implementation group: 'org.slf4j', name: 'slf4j-api'
+    implementation group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
+    implementation group: 'org.rocksdb', name: 'rocksdbjni', version: '7.9.2'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.10'
+    implementation group: 'com.google.code.externalsortinginjava', name: 'externalsortinginjava', version: '0.6.1'
+    implementation group: 'com.google.guava', name: 'guava', version: '31.1-jre'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
+    implementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
+    implementation group: 'joda-time', name: 'joda-time', version: '2.12.1'
+    implementation group: 'io.netty', name: 'netty-all', version: '4.1.85.Final'
+    implementation group: 'net.razorvine', name: 'pickle', version: '1.3'
+    implementation group: 'org.python', name: 'jython-standalone', version: '2.7.3'
+    implementation group: 'com.amazonaws', name: 'amazon-kinesis-client', version: '1.14.8'
+    implementation group: 'io.dropwizard.metrics', name: 'metrics-core', version: '4.2.13'
+    implementation group: 'io.dropwizard.metrics', name: 'metrics-jvm', version: '4.2.13'
+    implementation group: 'io.dropwizard.metrics', name: 'metrics-graphite', version: '4.2.13'
+    implementation 'jakarta.servlet:jakarta.servlet-api:5.0.0'
+    implementation 'org.eclipse.jetty:jetty-server'
+//    implementation 'org.mockito:mockito-core:4.11.0'
+
+    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
+    testImplementation group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
+
+    implementation (group:"org.springframework.boot", name:"spring-boot-starter-logging", version:"3.0.2") {
         exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
         exclude group: 'org.apache.logging.log4j', module: 'log4j-api'
     }
-    compile group:"org.springframework.boot", name:"spring-boot-starter-web", version:"2.7.5"
-    compile group:"org.springframework.boot", name:"spring-boot-starter-jetty", version:"2.7.5"
+    implementation group:"org.springframework.boot", name:"spring-boot-starter-web", version:"3.0.2"
```

Comment:
Shouldn't need a version on dependencies when using the spring boot plugin.

Reply:
unfortunately it is needed here.. I think it works for project dependencies but not for buildscript dependencies.